src/Apps/Order/__tests__/OrderApp.jest.tsx

```diff
@@ -678,7 +678,7 @@ describe("OrderApp", () => {
       mockGetENV.mockImplementation(() => ({ SALESFORCE_CHAT_ENABLED: true }))
       const props = getProps()
       const subject = getWrapper({
-        props: { ...props, location: { query: { isModal: true } } },
+        props: { ...props, match: { location: { query: { isModal: true } } } },
```

Comment:
Code in the component is pulling from `match`, so not sure how this ever passed before 🤷

Reply:
wow, that's interesting!! Thank you for identifying it and fixing it!! 🙇‍♀️